src/lib/spinel/radio_spinel.cpp

```diff
@@ -1906,6 +1906,10 @@ uint64_t RadioSpinel::GetNow(void) { return (mIsTimeSynced) ? (otPlatTimeGet() +
 
 uint32_t RadioSpinel::GetBusSpeed(void) const { return GetSpinelDriver().GetSpinelInterface()->GetBusSpeed(); }
 
+uint32_t RadioSpinel::GetBusLatency(void) const { return GetSpinelDriver().GetSpinelInterface()->GetBusLatency(); }
```

Comment:
I think we can simply put the latency in `RadioSpinel` as member. Is there any specific reason to put it into the Spinel Interface? For RadioUrl parsing, we can add a new param into radio url and parse it in `Posix::Radio::Init`. And then call `RadioSpinel::SetBusLatency` from there. Thoughts?

Reply:
I proceeded with that approach to keep the implementation as close to `GetBusSpeed` as possible, because `otPlatRadioGetBusSpeed` behaves similarly to the ones implemented, but I'll take a look if it can be done the way you proposed, as you're right, we don't need to handle the bus latency in the interface,thanks!

Reply:
Moved bus latency to be handled in `RadioSpinel` thanks for suggestion